packages/documentation-v7/src/stories/components/table/table.stories.ts

```diff
@@ -0,0 +1,236 @@
+import type { Args, Meta, StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import { mapClasses } from '../../../utils';
+
+const meta: Meta = {
+  title: 'Components/Table',
+  render: renderTable,
+  parameters: {},
+  args: {
+    variant: [],
+    borderStyle: 'null',
+    caption: 'List of users',
+    captionPlacement: 'bottom',
+    alignment: 'align-top',
+    content: null,
+  },
+  argTypes: {
+    content: {
+      name: 'Content',
+      description: 'Hidden helper arg for dynamically switching the table body',
+      table: {
+        disable: true,
+      },
+    },
+    caption: {
+      name: 'Caption',
+      description: 'A description of the table, very useful for screen reader users.',
+      control: {
+        type: 'text',
+      },
+      table: {
+        category: 'Caption',
+      },
+    },
+    captionPlacement: {
+      name: 'Caption Placement',
+      description:
+        'Choose where to place the description of the table. If a visual description is not necessary, it can also be hidden visually and still be of value for screen reader users.',
+      control: {
+        type: 'select',
+        labels: {
+          bottom: 'Bottom (default)',
+          top: 'Top',
+          hidden: 'Visually hidden',
+        },
+      },
+      options: ['bottom', 'top', 'hidden'],
+      table: {
+        category: 'Caption',
+      },
+    },
+    borderStyle: {
+      name: 'Border style',
+      description: 'Border style',
+      control: {
+        type: 'radio',
+        labels: {
+          'null': 'Default (underlined)',
+          'table-bordered': 'Bordered',
+          'table-borderless': 'Borderless',
+        },
+      },
+      options: ['null', 'table-bordered', 'table-borderless'],
+      table: {
+        category: 'General',
+      },
+    },
+    variant: {
+      name: 'Variants',
+      description: 'Stylistic table variants.',
+      control: {
+        type: 'check',
+        labels: {
+          'table-striped': 'Striped',
+          'table-hover': 'Hover enabled',
+        },
+      },
+      options: ['table-striped', 'table-hover'],
```

Comment:
It's not documented yet, but tables can also be used with the class `table-sm` to show them more compact.

```suggestion
        labels: {
          'table-sm': 'Small',
          'table-striped': 'Striped',
          'table-hover': 'Hover enabled',
        },
      },
      options: ['table-sm', 'table-striped', 'table-hover'],
```